src/masternodes/mn_checks.cpp

```diff
@@ -1211,6 +1211,12 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         if (!order)
             return Res::Err("order with creation tx " + makeoffer.orderTx.GetHex() + " does not exists!");
 
+        if (static_cast<int>(height) < consensus.EunosPayaHeight && makeoffer.expiry < CICXMakeOffer::DEFAULT_EXPIRY)
```

Comment:
Can we improve readability here a bit please, perhaps, by separating expiry out: 

```
        auto expiry = static_cast<int>(height) < consensus.EunosPayaHeight 
        ? CICXMakeOffer::DEFAULT_EXPIRY : 
        CICXMakeOffer::EUNOSPAYA_DEFAULT_EXPIRY;

        if (makeoffer.expiry < expiry) {
            return Res::Err("offer expiry must be greater than %d!", expiry - 1);
        }
```

Easier to reason with.

Reply:
Yeah, np. I did it like this somewhere else, but this was a quick fix so I rushed it little bit ;)

Reply:
Haha, thank you! I think it's time we started slowly improve on the code base :) Once checked-in, it's very difficult to get any thing changed!